_docker/drupal/Dockerfile

```diff
@@ -3,6 +3,7 @@ MAINTAINER Jason Porter <jporter@redhat.com>
 
 ARG http_proxy
 ARG https_proxy
+ARG composer_profile
```

Comment:
This could be defaulted to `production` i.e. `ARG composer_profile="production"` and then the only place it needs to be overridden is in the `_docker/drupal-dev/docker-compose.yml`

Reply:
It looks like he's set it up in the docker-composer files, but either way works. I guess I'd lean more toward being explicit, but it isn't a big deal either way.

Reply:
I haven't used this environment, but should the drupal-dev-local-dcp also use the 'developer' profile?

Reply:
@jordanpagewhite Yes.